src/examples/js/CartClientExample.js

```diff
@@ -72,6 +72,15 @@ console.log(
     await cartClientExample.completeCartById('YOUR_CART_ID', {
       // YOUR_PAYMENT_METHOD can be one of COD or PREPAID
       paymentMethod: 'YOUR_PAYMENT_METHOD',
+      discountCode: 'YOUR_DISCOUNT_COUPON_CODE',
+      shippingAddress: {
+        address1: 'YOUR_CUSTOMER_ADDRESS',
```

Comment:
Let's write optional fields as well here and add a comment that they are optional

Reply:
For shipping Address required fields are address1, city, , firstName, lastName, phone, zip. Let's write required fields only, because in Address object there is lot of fileds